packages/components/status/src/status.js

```diff
@@ -0,0 +1,46 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './status.css';
+import { STATUS_TYPE } from './utils/constants';
+
+customElementDefineHelper(
+	'ts-status',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				status: { type: String, reflect: true },
```

Comment:
Consider to add the `dir` property to support Right-To-Left. Also add a fallback to a body `dir`. You can use the `header` component as an example.

Reply:
done